mne/beamformer/_compute_beamformer.py

```diff
@@ -123,10 +123,13 @@ def _check_one_ch_type(info, picks, noise_cov):
     info['comps'] = comps
     ch_types =\
         [_contains_ch_type(info_pick, tt) for tt in ('mag', 'grad', 'eeg')]
-    if sum(ch_types) > 1 and noise_cov is None:
+    if beamformer == 'lcmv' and sum(ch_types) > 1 and noise_cov is None:
         raise ValueError('Source reconstruction with several sensor types '
                          'requires a noise covariance matrix to be '
                          'able to apply whitening.')
+    elif beamformer == 'dics' and sum(ch_types) > 1:
+        warn('The use of several sensor types with the DICS beamformer is '
+             'not heavily tested yet.')
```

Comment:
This will `warn` even for mag+grad, right? This doesn't seem right since @wmvanvliet's pipeline was developed on Neuromag systems. It might even be tested on mag+grad+eeg already, not sure.

Reply:
Hmm, then we have a problem because we use mag+grad in our example:

https://github.com/mne-tools/mne-python/blob/master/examples/inverse/plot_dics_source_power.py#L39

We should probably not advertise something like this as the primary DICS example that isn't known to work properly yet. If we want to keep both types in there, we should probably add a `.. note` or `.. warning` to the example as well as the `make_dics` docstring that results will be skewed toward the sensor type with larger values.

Okay with you @wmvanvliet ?

Reply:
ping @wmvanvliet

Reply:
I agree totally that examples throwing warnings like that are not ideal and pointed that out some time ago in this thread already ...